code/modules/surgery/organs/internal/stomach/_stomach.dm

```diff
@@ -312,15 +322,25 @@
 	emp_vulnerability = 20
 	metabolism_efficiency = 0.1
 
-/obj/item/organ/internal/stomach/cybernetic/emp_act(severity)
-	. = ..()
-	if(. & EMP_PROTECT_SELF)
+/obj/item/organ/internal/stomach/cybernetic/surplus
+	name = "surplus prosthetic stomach"
+	desc = "A mechanical plastic oval that utilizes sulfuric acid instead of stomach acid. \
+		Very fragile, with painfully slow metabolism.\
+		Offers no protection against EMPs."
+	icon_state = "stomach-c-s"
+	maxHealth = STANDARD_ORGAN_THRESHOLD * 0.35
+	emp_vulnerability = 100
+	metabolism_efficiency = 0.025
+
+//surplus organs are so awful that they explode when removed, unless failing
+/obj/item/organ/internal/stomach/cybernetic/surplus/on_surgical_removal(mob/living/user)
+	if(organ_flags & (ORGAN_FAILING|ORGAN_EMP))
```

Comment:
Could make this an element rather than copying it across. 
I don't super care but it would be nice.

Reply:
In hindsight I agree, will do it when possible